test-locales.js

```
@@ -0,0 +1,23 @@
+assert(languageCodes.length > 0, 'languages object should not be empty');
+
+// Check that names are simplified (no semicolons or technical notes like "(1453-)")
+// Note: countryDisplayName adds " (Country Name)", so we check the language portion
+for (const code of languageCodes) {
+	const displayName = languages[code];
+	assert(typeof displayName === 'string', `Display name for ${code} should be a string`);
+	assert(!displayName.includes(';'), `Display name for ${code} should not contain a semicolon: "${displayName}"`);
+	
+	// Check specifically for common technical markers that should be filtered out
+	assert(!displayName.includes('1453'), `Display name for ${code} should not contain technical years: "${displayName}"`);
+}
```

ESLint

[error] Newline required at end of file but not found.

(eol-last)